docs/dev/table/functions/systemFunctions.md

```diff
@@ -4884,12 +4884,18 @@ MAP.at(ANY)
     <tr>
       <td>
 {% highlight text %}
-ROW(value1, [, value2]*)
-(value1, [, value2]*)
+-- implicit constructor with parenthesis
+(value1 [, value2]*)
+
+-- explicit ROW constructor 
+ROW(value1 [, value2]*)
 {% endhighlight %}
       </td>
       <td>
         <p>Returns a row created from a list of values (<i>value1, value2,</i>...).</p>
+        <p>Note: The implicit row constructor supports arbitrary expressions as fields but requires
+        at least two fields. The explicit row constructor can deal with an arbitrary number of fields
+        but does not support all kinds of field expressions well currently.</p>
```

Comment:
nit: Maybe we could link to the JIRA issues targeted at fixing it? I am a bit afraid we might forget to update this section when we fix the issue properly. With a link there is a slightly better chance users will be able figure it out if it was fixed.